ironfish-cli/src/commands/accounts/repair.ts

```diff
@@ -156,15 +158,23 @@ export default class Repair extends IronfishCommand {
         }
 
         if (!isExpired) {
-          unconfirmedBalance += decryptedNoteValue.note.value()
+          const assetIdentifier = decryptedNoteValue.note.assetIdentifier()
+          const unconfirmedBalance = unconfirmedBalances.get(assetIdentifier) ?? BigInt(0)
+          unconfirmedBalances.set(
+            assetIdentifier,
+            unconfirmedBalance + decryptedNoteValue.note.value(),
+          )
         }
       }
     }
 
     this.log(
```

Comment:
We should move this into the loop below to output for each asset that it updates

Reply:
Done